hw/ip/pwrmgr/pwrmgr_pkg.core

```diff
@@ -8,7 +8,7 @@ description: "Power manager package"
 filesets:
   files_rtl:
     depend:
-      - lowrisc:ip:pwrmgr_reg
+      - lowrisc:systems:pwrmgr_reg
```

Comment:
Can you try undoing this change?

Reply:
That causes all the dv environment to break: the ip and autogen pwrmgr_reg code is not compatible, in fact the ip is missing a CSR altogether, and some parameterizations are incorrect.

I think the real problem is that pwrmgr_pkg depends on pwrmgr_reg. Seems completely backwards: packages should not depend on RTL. The problem probably from the dependency of pwrmgr_pkg on things like pwrmgr_reg_pkg::NumWkups. I think NumWkups and NumRstReqs should be moved to pwrmgr_pkpg to lop the dependency.